.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/roman-numerals-frontend:latest
+
+  deploy:
+    needs: build
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Install Docker Compose
+        run: sudo apt-get install docker-compose-plugin
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)